SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -2198,6 +2198,29 @@ class CFVMFlowSolverBase : public CSolver {
       Inlet_FlowDir[val_marker][val_vertex][val_dim] = val_flowdir;
   }
 
+  /*!
+   * \brief Set a component of the farfield velocity vector.
```

Comment:
This description is not very accurate, all the components are set based on aoa and aos